### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Nolram12345** [Whinarn/UnityMeshSimplifier#49]
This package, while it seems very capable, has absolutely no documentation from what I have found. I would really like to use it, but I have no idea how. ANY examples or documentation would be needed greatly.

**Comment by Whinarn** [comment]
You are right in that there is currently no usable documentation for this package as of now.
Originally it was intended for programmers and had a very simple API that didn't need much of an explanation. But with the introduction of the `LOD Generator Helper` component, there is a greater need for documentation.

If you are a programmer then the API is pretty well documented with XML-comments with [these small examples](https://github.com/Whinarn/UnityMeshSimplifier#how-to-use) enough to get you started, but if not, then you are out of luck.
I have put it on hold since it seems like most people have been able to use it anyways, but I'll see if I can make time to add some basic documentation to the repository wiki or something. While I'm not very interested in developing further features, it feels like I should at least add some basic documentation before I can consider this project "completed".

**Comment by Nolram12345** [comment]
Ah, thanks for the clarification. Sad that you have more or less shelved this now - there would have probably been some interesting places to go with this, like dynamic GPU mesh decimation. But if you could provide any more documentation that would be very appreciated.

**Comment by Whinarn** [comment]
Absolutely, there are plenty of features that would improve this package. Unfortunately however I don't have the time or interest to maintain it more than fixing bugs (and potentially documentation). Since I am not a user of the package myself it's hard to motivate why I should spend time working on it, and it's also more difficult to be proactive with it. Obviously I'm a supporter of open source, but I'd rather work on something that is better aligned with my interests and expertise.

I hope that someone someday can take over maintaining this package, or create another free and open source alternative for everyone out there not looking to pay $100 for their hobby or indie game.

But we'll see what happens. I might decide to pick this up again if I get back into game development.

I cannot promise any documentation this or next week however. Just a heads up.

**Comment by Nolram12345** [comment]
Ah, that is absolutely understandable, hope someone else can be found...
Also, I am a bit unsure whether you fully understood what I meant with lacking documentation - it would also be great to have more documentation on the actual usage of the API as well as the LOD Generator component.

**Comment by Whinarn** [comment]
Hi @Nolram12345 ,

I just wanted to let you know that I have worked a bit to document the `LOD Generator Helper` component here: https://github.com/Whinarn/UnityMeshSimplifier/wiki/LOD-Generator-Helper-component

Please have a look if you have the time.

I will leave this issue open since there are much more things that can be documented.
